core/src/main/resources/jenkins/model/BuiltInNodeMigration/message.properties

```diff
@@ -0,0 +1,40 @@
+# The MIT License
+#
+# Copyright (c) 2021 Daniel Beck
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+blurb = The word "master" is being retired as the term for the main Jenkins process and the built-in node. \
+  The main process is now called "controller" and the built-in node is called just "built-in node". \
+  The UI has been updated with these changes. \
+  The following features are also affected: \
+  <ul>\
+    <li>The implicit label of the built-in node changes from <code>master</code> to <code>built-in</code>.</li>\
+    <li>The built-in node''s <code>NODE_NAME</code> environment variable also changes from <code>master</code> to <code>built-in</code>.</li>\
+  </ul>\
+  These changes could affect build behavior, so are not applied automatically. \
+  Before you apply these changes, you should do the following: \
+  <ul>\
+    <li>Review label assignments in job configurations and tool installers for uses of <code>master</code> label. \
+        Any such label assignments will not match the built-in node after migration. \
+        Besides updating these assignments, you could also explicitly add the <code>master</code> label to the built-in node. \
+  </li>\
+    <li>Review use of the <code>NODE_NAME</code> environment variable in build scripts.</li>\
+  </ul> \
+  <a href="https://www.jenkins.io/redirect/built-in-node-migration" rel="noopener noreferrer" target="_blank">Learn more.</a>
```

Comment:
I can't see a PR for this?

Reply:
Still local only, had some trouble with Docker yesterday. But yes, this will have a linked PR.

Reply:
https://github.com/jenkins-infra/jenkins.io/pull/4496